src/backend/internal.py

```diff
@@ -0,0 +1,31 @@
+import abc
+from typing import List
+
+from docarray import DocumentArray
+import numpy as np
+import tensorflow as tf
```

Comment:
Tensorflow or Pytorch as the first backend? Or is this just an example?

Reply:
this import shouldnt be here - thanks